minigalaxy/download_manager.py

```diff
@@ -8,6 +8,17 @@
 
 
 class __DownloadManger:
+    #TODO: Move static method to ui.gametile?
```

Comment:
It is ok where it is. I would keep only UI stuffs in ui.gametile.
If you don't like this static method you can move it out of __DownloadManager() class as function not related to this class.